odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+standard_library.install_aliases()
+
+import numpy as np
+from odl.trafos import FourierTransform
+
+
+__all__ = ('fbp_op',)
+
+
+def fbp_op(ray_trafo):
+    """Create Filtered BackProjection from a ray transform.
+
+    Parameters
+    ----------
+    ray_trafo : `RayTransform`
+
```

Comment:
Perhaps add some options for users to specify one of the commonly used filters here.